src/components/cart/CartItem.jsx

```diff
@@ -1,41 +1,51 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
 import Button from '@mui/material/Button';
-import Box from '@mui/material/Box';
-import { useGlobalContext } from '../../context';
-//cannot distructure id of tiem
-const CartItem = ({ item, addToCart }) => {
-  const { remove } = useGlobalContext();
-  //need to pull in global context stuff {increase, decrease, toggleAmount}
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
 
-  const { id, title, price, amount, image } = item;
+import { useApi } from '../../hooks/useApi';
 
+const CartItem = ({ id }) => {
+  const [cartItem, setCartItem] = useState('');
+  const request = useApi();
+  const getProduct = async () => {
```

Comment:
We're already sending the entire product from the cart screen, let's just refactor to use the existing data instead of making new network requests